test/dotnet.Tests/GivenThatTheUserIsRunningDotNetForTheFirstTime.cs

```diff
@@ -78,19 +78,75 @@ public void ItShowsTheAppropriateMessageToTheUser()
         }
 
     	[Fact]
-    	public void ItRestoresTheNuGetPackagesToTheNuGetCacheFolder()
+    	public void ItCreatesASentinelFileUnderTheNuGetCacheFolder()
     	{
             _nugetCacheFolder
                 .Should()
                 .HaveFile($"{GetDotnetVersion()}.dotnetSentinel");            
     	}
 
         [Fact]
-        public void ItCreatesASentinelFileUnderTheNuGetCacheFolder()
+        public void ItRestoresTheNuGetPackagesToTheNuGetCacheFolder()
         {
             _nugetCacheFolder
                 .Should()
                 .HaveDirectory("microsoft.netcore.app");
+
+            _nugetCacheFolder
```

Comment:
I wonder if we should make this a list and add a Should().HaveDirectories() to the DirectoryInfoAssertions, if there isn't one already.